libsolidity/ast/AST.cpp

```diff
@@ -204,6 +204,17 @@ vector<Declaration const*> const& ContractDefinition::inheritableMembers() const
 
 		for (StructDefinition const* s: definedStructs())
 			addInheritableMember(s);
+
+		for (EnumDefinition const* e: definedEnums())
+		{
+			solAssert(e, "definedEnums() contained a null pointer.");
+			addInheritableMember(e);
+			for (ASTPointer<EnumValue> member: e->members())
+			{
+				solAssert(member, "EnumDefinition::members() contained a null pointer.");
```

Comment:
Wouldn't it be better to move this inside the `addInheritableMember` function? I know the message will lose some information but the compiler is very easy to debug.